### PR TITLE
Feat/pdf from server

### PR DESCRIPTION
**Pull request by jona159**
- fetch pdf from server
- render pdf in embed
- remove PDF creation from the frontend

On the backend use: ```feat/generate-pdf```

### CODE REVIEW

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts [hunk 1]
```diff
@@ -61,12 +67,17 @@ export class ExportPdfDialog extends BaseDialog {
...
+		this.pdfService.getPdf(badge.slug).subscribe((url) => {
+			this.pdfSrc = url;
+		})
 
-		return new Promise<void>((resolve, reject) => {
-			this.resolveFunc = resolve;
-			this.rejectFunc = reject;
-		});
+		
+		// this.generateSingleBadgePdf(this.badge, markdown);
```
Comment (YouQam) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:75]: Add a comment here to show this is the old implementation or remove it if no longer used.

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts [hunk 2]
```diff
@@ -304,10 +315,10 @@ export class ExportPdfDialog extends BaseDialog {
 		}
 
 		const badge_image = await this.convertImageToDataURL(badgeClass.image);
-		const badge_image_aspectRatio = await this.getAspectRatio(badge_image);
+		// const badge_image_aspectRatio = await this.getAspectRatio(badge_image);
```
Comment (YouQam) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:318]: here as well.

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts [hunk 3]
```diff
@@ -304,10 +315,10 @@ export class ExportPdfDialog extends BaseDialog {
...
 
 		const issuer_image = badgeClass.issuer.image ? await this.convertImageToDataURL(badgeClass.issuer.image) : null;
-		const issuer_image_aspectRatio = issuer_image ? await this.getAspectRatio(issuer_image) : null;
+		// const issuer_image_aspectRatio = issuer_image ? await this.getAspectRatio(issuer_image) : null;
```
Comment (YouQam) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:321]: here also.

File: src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts [hunk 4]
```diff
@@ -465,18 +476,18 @@ export class ExportPdfDialog extends BaseDialog {
...
-							this.doc.addImage(
-								issuer_image,
-								'PNG',
-								pageWidth / 2 - 15,
-								yPos - 2,
-								30,
-								issuer_image_height,
-							);
-						}
+						// if (issuer_image) {
```
Comment (YouQam) [inline src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts:479]: here also.

**Review by YouQam — COMMENTED** [review]
Fix the build error then I'll approve it.

**Review by umut0 — APPROVED** [review]

**Review by YouQam — APPROVED** [review]

**Pull request merged**